src/main/java/ru/ayaz/MessageProcessor.java

```diff
@@ -0,0 +1,28 @@
+package ru.ayaz;
+
+public class MessageProcessor {
+
+    public boolean isCommand(String message) {
+        message = message.trim();
```

Comment:
It is discouraged to change function parameters (even if only reference and not the underlying object is changed). Might consider creating a local variable called trimmedMessage or even doing the whole function logic in one line (I would prefer this actually in this case).